installation_policies/pre_spack_install_policies.md

```diff
@@ -0,0 +1,124 @@
+
+    2. If the compilers and/or flags are not explicitly set on input but are set in the environment variables
+    `FC`, `CC`, `CXX`, `CPP`<sup>6</sup>, `FFLAGS`, `FCFLAGS`, `CFLAGS`, `CXXFLAGS`, `CPPFLAGS`,<sup>7</sup>
+    `LDFLAGS`, then the compilers and flags must be set to these.  If both `FFLAGS` and `FCFLAGS` are set, then they
+    need to be the same or it is an error.
+
+    3. If the compilers and/or compiler flags are not explicitly passed in or defined in the environment variables listed 
+    above, then the package is free to try to find compilers on the system and set the compiler flags consistent with the 
+    other settings defined below (e.g., shared libraries vs. static libraries, debug vs. non-debug).<sup>8</sup>
+
+<sup>5</sup> Packages must support using the MPI compiler wrappers for these arguments.
+
+<sup>6</sup> The environment variable CPP is not supported by raw CMake.
+
+<sup>7</sup> The environment variable CPPFLAGS is not supported by raw CMake.
+
+<sup>8</sup> All CMake projects should use the same built-in CMake algorithm to find the default compilers, so even when no 
+explicit compilers or flags are set they should use the same compilers and flags.  Also, raw CMake projects will append 
+compiler flags based on the build type.  See "Selecting compiler and linker options".
+
+----
+
+4. Create libraries with debugging information and possible additional error checking (default is debug in XSDK mode).
+
+    1. `--dis/enable-debug` 
+
+    2. `CMAKE_BUILD_TYPE=[Debug,Release]`
+
+----
+
+5. Select option used for indicating whether to build shared libraries (default is shared in XSDK mode).
+
+    1. `--dis/enable-shared (configure)` 
+
+    2. `BUILD_SHARED_LIBS=[YES,NO]` 
+
+----
+
+6. Build interface for a particular additional language.
+
+    1.  `--dis/enable-<language>` 
+
+    2. `XSDK_ENABLE_<language>=[YES,NO]` 
+
+
+7. Determine precision for packages that build only for one precision (default is double). Packages that handle all 
+precisions automatically are free to ignore this option.
+
+    1. `--with-precision=[single,double,quad]`
+
+    2. `XSDK_PRECISION=[SINGLE,DOUBLE,QUAD]` 
+
+----
+
+8. Determine index size for packages that build only for one index size (default is 32). Packages that handle all index sizes
+automatically are free to ignore this option.
+
+    1. `--with-index-size=[32,64]`
+
+    2. `XSDK_INDEX_SIZE=[32,64]`
+
+----
+
+9. Set location of BLAS and LAPACK libraries (default is to locate one on the system automatically). 
+
+    1. `--with-blas-lib="linkable list of libraries” --with-lapack-lib=”linkable list of libraries”` -- it is fine to use
+    -L and -l flags in the lists
+
+    2. `TPL_BLAS_LIBRARIES="linkable list of libraries”`, `TPL_LAPACK_LIBRARIES=”linkable list of libraries”` -- should not use
```

Comment:
The CMake default is `BLAS_LIBRARIES` when using the `find_package(BLAS REQUIRED)`.

Why reinvent the wheel? Find BLAS is build into CMake, why not use it?

Reply:
@mkstoyanov  See my comment above about this file being an archive of the current installation policies.